packages/amplify-migration-tests/src/configure_tests.ts

```diff
@@ -9,6 +9,8 @@ import { amplifyConfigure as configure, isCI, installAmplifyCLI, injectSessionTo
 async function setupAmplify(version: string = 'latest') {
   // install CLI to be used for migration test initial project
   await installAmplifyCLI(version);
+  
+  console.log("INSTALLED CLI:", version);
```

Comment:
sanity check :)

Reply:
nit: should we also print result of amplify version ? to verify that installAmplifyCLI actually installs requested version.